10.0/s2i/bin/run

```diff
@@ -1,2 +1,12 @@
 #!/bin/bash
-exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+run_wildfly() {
+  if [ "$DEV_MODE" == true ]; then
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
+  else
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
```

Comment:
rather than this if-else i think i'd prefer you just update the standalone.xml to contain:

`<deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="250" auto-deploy-zipped="${DEV_MODE}" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>`
        
and in the run script:
DEV_MODE=${DEV_MODE:-false}